analysis_scripts/qc_scripts/bg_nature_qc.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "bg_nature_qc"
+author: "Natalie Davidson"
+date: "3/29/2021"
+output: rmarkdown::github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+require(data.table)
+require(here)
+require(ggplot2)
+require(stringr)
+
+
+proj_dir = here()
+source(file.path(proj_dir, "/utils/scraper_processing_utils.R"))
+source(file.path(proj_dir, "/process_doi_data/springer_scripts/springer_scrape_utils.R"))
+
+```
+
+
+## Overview
+
+This notebook will QC the scraped author information from nature research articles + letters.
+This dataset is a background dataset used to compare gender + country rates in Nature News.
+This analysis looks at 3 steps of a pipeline
+
+1. raw scraped data: `/data/author_data/downloads`
+
+2. gender predictions : `/data/author_data/nature_author_gender.tsv`
+
+3. location predictions from Springer API response: `/data/author_data/all_author_country.tsv`
+
+
+## Pipeline Step 1: Author Information Scrapes
+
+### Read in the scraped info
+
+```{r step1_read, echo=TRUE, warning=FALSE, message=F}
+
+# read in the scraped citations from nature news articles for each year
+pipeline_1_dir = paste(proj_dir,
```

Comment:
Very minor: I think `paste0` lets you do the same without the need for the `sep` argument. Or is it different?

Reply:
And you were using `file.path` in other parts like this I think.

Reply:
yes, I should be using file.path. an old habit to keep using paste

Reply:
fixed in: https://github.com/greenelab/nature_news_disparities/pull/19/commits/d2f65cfa1ebc94522281e63ab529fe48c0cf1bb4